server/routes/serviceProviderReferrals/interventionProgressView.ts

```diff
@@ -34,7 +34,7 @@ export default class InterventionProgressView {
   }
 
   private actionPlanSummaryListArgs(tagMacro: (args: TagArgs) => string, csrfToken: string): SummaryListArgs {
-    const rows: SummaryListRow[] = [
+    const rows: SummaryListArgsRow[] = [
```

Comment:
Was this supposed to be part of this commit? Is this a new type that was generated that we wrote wrong, originally? Would you mind just adding a quick line about this change to the commit message if so?

Reply:
Yeah, it's just that the name of the type that we were using before isn't consistent with the naming used in the generated types, so I had to change it to fix the compilation error. I'll mention in commit message.